docs/components/_index.scss

```diff
@@ -1,6 +1,8 @@
 /// load styles required to demonstrate each component
 @import "box/index";
 @import "grid/index";
+@import "icon/demo";
+@import "navigation/demo";
```

Comment:
Please rename file reference to `index`.

Reply:
Changes have been done, please review